internal/repo/datasource/sql_test.go

```diff
@@ -0,0 +1,357 @@
+package datasource
+
+import (
+	"errors"
+	"github.com/DATA-DOG/go-sqlmock"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/vatsal278/TransactionManagementService/internal/config"
+	"github.com/vatsal278/TransactionManagementService/internal/model"
+	"reflect"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestSqlDs_HealthCheck(t *testing.T) {
+	if testing.Short() {
```

Comment:
All test cases were supposed to be mocked. Why is this skipped?

Reply:
done